scripts/sync-bottle-hashes.sh

```diff
@@ -0,0 +1,55 @@
+#! /usr/bin/env bash
+# SPDX-FileCopyrightText: 2021 TQ Tezos <https://tqtezos.com/>
+#
+# SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+# This script takes the new tag and the OS name as its arguments
+# and handles git shenanigans related to syncing two pipelines over a common branch.
+
+set -e
+
+if [ -z "$1" ] || [ -z "$2" ]; then
+    echo "Please call this script with the release tag and the OS name."
+else
```

Comment:
Nitpick, but since there `set -e` above, this should work as well:
```suggestion
if [ -z "$1" ] || [ -z "$2" ]; then
    echo "Please call this script with the release tag and the OS name."
    exit 1
fi
```
and avoid the `else` and the resulting indentation (plus it returns an error code).